source/leadership.rst

```diff
@@ -0,0 +1,133 @@
+.. _leadership:
+
+##################
+Project Leadership 
+##################
+
+yt-project Members
+------------------
+
+A member is someone who has made continued and significant contribution to the
+project (changes to the codebase, discussion on mailing lists, feedback on pull
+requests, documentation, teaching, etc.) for some period of time. After such a
+period, potential new members are nominated for membership by an existing
+member and confirmed by positive votes from three additional members. Once a
+developer becomes a member, they remain a member for life. A member maintains
+the option to give up their membership and have their name removed from the
+list. Membership may be revoked for anyone who is deemed to be directly harmful
+to the project or the community upon a nomination by another member and five
+supporting member votes and an additional vote by the steering committee.
+Members may also elect to become emeritus members, and be recognized for their
+contributions to the project while also rescinding their write access to the
+project. 
+
+Upon the initial creation of yt membership status, all
+developers that had contributed at least 50 changesets were granted
+membership. The `Project Members page <https://yt-project.org/members.html>`_ 
+gives a description of the requirements
+for membership and provides a list of all members and the year in which
+membership was granted. Members have write access to all official yt
+repositories and can, therefore, accept and merge pull requests.
+
+Subproject Teams
```

Comment:
Just a suggestion, but maybe instead of leadership roles for each subproject, we could identify areas of expertise and allow people to tag themselves with as many as they are comfortable with. This would denote them as points of contact for those areas. I don't think this document should be changed yet, but I'm in favor of discussing it.

Reply:
After the yt team meeting we decided to eliminate subproject leaders and deputies. Do you think this alternative is an avenue that we want to explore? We could add a points-of-contact section in the governance or on the project webpage for what you've suggested.

Reply:
I'm going to mark this as resolved and we can add points-of-contact later if we feel it's necessary.